CMakeLists.txt

```diff
@@ -217,6 +217,7 @@ add_library(mixxx-lib STATIC EXCLUDE_FROM_ALL
   src/controllers/keyboard/keyboardeventfilter.cpp
   src/controllers/learningutils.cpp
   src/controllers/midi/midicontroller.cpp
+  src/controllers/midi/midicontrollerpreset.cpp
```

Comment:
Also update build/depends.py until we remove SCons support

Reply:
Whoops, sorry. I'll fixup the commit.